config.go

```diff
@@ -141,7 +141,7 @@ func (c Configs) appPaths() ([]string, []string) {
 		warnings = append(warnings, fmt.Sprintf("Both .aab and .apk files provided, using the .aab file(s): %s", strings.Join(aabs, ",")))
 	}
 
-	if len(aabs) > 1 {
+	if len(aabs) > 1 && aabs != nil {
```

Comment:
aabs can not be nil if len(aabs) > 1, so this check is dead code. len(nil) is 0: https://golang.org/src/builtin/builtin.go?s=5727:5747#L145

Reply:
Removed.